README.md

```diff
@@ -10,4 +10,5 @@ Describe how you approached to problem, and what tools and techniques you used t
 
```

Comment:
It would be great to be able to read some more about your project, what your problems were and how you solved it. Also what you would have focused on if you had more time?

Reply:
Thanks Joanna &  Venessa.
I forgot to change the text in README.
I will update ASAP